readme.md

```diff
@@ -55,6 +55,7 @@ const route = fakeServer.http
     .get()  // Http Method (mandatory). See Supported HTTP Methods section.
     .to(pathRegex) // Route Path (mandatory). May be regex
     .withBody(object) // Route Restriction (optional). See Route Restrictions section.
+    .withDelay(ms) // Delay response in ms (optional)

```

Comment:
not related to this line, but please update the `Route Restrictions` section that several restrictions are now supported.

Reply:
It's not supported